### ISSUE TITLE
Avatar availability "bulbs" or badges not accessible

### ISSUE THREAD

**Issue by dialpad-drew** [dialpad/dialtone#239]
They rely on color alone to communicate their meaning; we should strengthen that by adding different shapes or glyphs for color blind or low vision users.

Green = available
Orange = Busy. Eg. In a meeting.
Red = Busy (I think?)

![image](https://user-images.githubusercontent.com/77751297/112335267-b5945880-8c81-11eb-95eb-8009418e4baa.png)
